### PR TITLE
Fix a typo

### PR DESCRIPTION
**Pull request by houseroad**
I was about to add DepthToSpace support. However it turns out not trivial. This PR is a byproduct... sigh.

### CODE REVIEW

**Comment by houseroad** [comment]
@caffe2bot retest this please

**Review by Maratyszcza — APPROVED** [review]
LG, thanks!

File: caffe2/onnx/backend.cc [hunk 1]
```diff
@@ -332,11 +332,11 @@ Caffe2Backend::get_special_operators() const {
       unordered_map<std::string, Caffe2Backend::SpecialOpConverter>
           kSpecialOperators = {
               {"Constant", &Caffe2Backend::CreateConstant},
-              {"Conv", &Caffe2Backend::CreateConvePoolOpBase},
-              {"AveragePool", &Caffe2Backend::CreateConvePoolOpBase},
-              {"GlobalAveragePool", &Caffe2Backend::CreateConvePoolOpBase},
-              {"GlobalMaxPool", &Caffe2Backend::CreateConvePoolOpBase},
-              {"MaxPool", &Caffe2Backend::CreateConvePoolOpBase},
```
Comment (yinghai) [inline caffe2/onnx/backend.cc:339]: 😂

**Review by yinghai — APPROVED** [review]

**Comment by Yangqing** [comment]
(appveyor error seems irrelevant)

**Pull request merged**